apps/docs/src/content/components/status-icon.mdx

```diff
@@ -0,0 +1,30 @@
+import { Callout } from "nextra/components";
+import {
+  statusIconClasses as classes,
+  HvStatusIcon,
+  statusIconClasses,
+} from "@hitachivantara/uikit-react-core";
+
+import { Header } from "../../components/Header";
+
+<Header
+  params={{ name: "StatusIcon", classes }}
+  Component={HvStatusIcon}
+  componentName="HvStatusIcon"
+  controls={{
+    variant: { defaultValue: "success" },
+    size: { defaultValue: "sm", type: "slider" },
+    type: { defaultValue: "full" },
+  }}
+/>
+
+## Custom icon
+
+```tsx live
+<div className="flex justify-start">
+  <HvStatusIcon
```

Comment:
We shouldn't have to add custom styles to get the "normal behaviour".
`HvStatusIcon` should be `inline-flex`, like `HvIconContainer`

```suggestion
<HvStatusIcon
```